plugins/BEdita/Core/src/Model/Action/AddAssociatedAction.php

```diff
@@ -44,7 +44,20 @@ protected function diff(EntityInterface $entity, array $relatedEntities)
         foreach ($relatedEntities as $relatedEntity) {
             $primaryKey = $relatedEntity->extract($bindingKey);
             if (in_array($primaryKey, $existing)) {
-                continue;
+                if (!($this->Association instanceof BelongsToMany) || !$relatedEntity->has('_joinData')) {
+                    continue;
+                }
+
+                $relatedEntity->_joinData = $this->Association->junction()->patchEntity(
```

Comment:
Scrutinizer complains and suggests... do we ignore him?

Reply:
I can use `$relatedEntity->set('_joinData', ...)` in place of `$relatedEntity->_joinData = ...`. Agreed!